docs/clients/server-side.md

```diff
@@ -30,6 +30,11 @@ https://github.com/Flagsmith/flagsmith-python-client
 
 https://github.com/Flagsmith/flagsmith-java-client
 
+</TabItem>
+<TabItem value="kotlin" label="Kotlin">
```

Comment:
I think we should advertise kotlin for server side applications. It doesn't behave like any of our other server side clients. We would like to encourage people to just add the Java client as a dependency for kotlin server applications. 

What do you think @dabeeeenster ?